test/extensions/bootstrap/wasm/config_test.cc

```diff
@@ -48,113 +67,69 @@ class WasmFactoryTest : public testing::TestWithParam<std::string> {
 };
 
 INSTANTIATE_TEST_SUITE_P(Runtimes, WasmFactoryTest,
-                         testing::Values("v8"
+                         testing::Values("v8",
 #if defined(ENVOY_WASM_WAVM)
```

Comment:
FYI this pattern (`#if` inside macro expansion) breaks MSVC

Reply:
@jplevyak Should we just nuke WAVM support?

Reply:
Yes.  I am going to do that as soon as I merge this.